src/lib/config.ts

```diff
@@ -525,16 +536,20 @@ export class default_config {
     }
 
     /**
-     * Whether to use the keytranslatemap in various maps.
+     * Whether to use the keytranslatemap
      */
-    keytranslatemodes: { [key: string]: "true" | "false" } = {
-        nmaps: "true",
-        imaps: "false",
-        inputmaps: "false",
-        ignoremaps: "false",
-        exmaps: "false",
-        hintmaps: "false",
-    }
+    usekeytranslatemap: "true" | "false" = "false"
+
+    /**
+     * Instead of fetching actual character which depends on selected layout,
+     * use machine code of a key. Makes any keyboard work as if it is qwerty.
+     * Recommended for everyone, especially multilingual.
+     *
+     * If you don't want qwerty, you can use keytranslatemap to map it to something else,
+     * and this setting is still useful because it makes all events tied to the physical
+     * keys.
+     */
+    alwaysqwerty: "true" | "false" = "true"
```

Comment:
I'm really not sure this should be true by default, what do other applications do?

If you've never seen a QWERTY keyboard in your life it must be pretty weird to be told to press the `w` key to open a new window and then find that your `w` key doesn't work

Reply:
[Here](https://github.com/tridactyl/tridactyl/pull/4439#issuecomment-1315772177) i propose hybrid mode idea.